Tests/test_file.py

```diff
@@ -701,7 +701,7 @@ def test_errors(self):
 
         with self.assertRaises(OSError) as cm:
             open('path_too_long' * 100)
-        self.assertEqual(cm.exception.errno, (36 if is_posix else 22) if is_netcoreapp and not is_posix or sys.version_info >= (3,6) else 2)
+        self.assertEqual(cm.exception.errno, (63 if is_osx else 36 if is_linux else 22) if is_netcoreapp and not is_posix or sys.version_info >= (3,6) else 2)
```

Comment:
Not that I mind the explicit number check, but wondering if we should be using the `errno` module values instead to make things easier to decipher.

Reply:
Good point, assuming that `errno` is not buggy 😄